### ISSUE TITLE
Dependency from unsecure repo error when using run on an ontology

### ISSUE THREAD

**Issue by msicilia** [phillord/tawny-owl#66]
I have just installed **leiningen** on Mac OS High Sierra 10.13.5 using `brew` and tried the following from the manual:
```
$ lein new ontology helloworld
$ cd helloworld/
$ lein run
```
And I get this message from the last command:

```
Tried to use insecure HTTP repository without TLS.
This is almost certainly a mistake; however in rare cases where it's
intentional please see `lein help faq` for details.
```
I have read the FAQ of **leiningen** and it mentions this is due to a dependency configured to download  from a repository that does not use TLS encryption. Not sure if this is intentional or unavoidable, just wanted to ask before forcing download from an unsecure repo.

Thanks in advance.

**Comment by msicilia** [comment]
Many thanks for the reply. Knowing that all the dependencies were ok, I just tried to reinstall everything, as I found reports of similar errors searching the Web in different projects (not Tawny-OWL). 

Now it worked, after I manually removed the "`.m2`" folder containing the Maven repository. Maybe it was a problem with a previous install or some error with incomplete downloads that was making `lein` give that strange error. It seems a Maven error, but happily I have it now fixed.
